src/components/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react';
+
+import Cards from '../Cards/Cards';
+import InPlay from '../InPlay/InPlay';
+import Button from '../Button/Button';
+import PopUp from '../_commonComponents/PopUp/PopUp';
+
+import { getCards } from '../../constants/preparation';
+
+const cardList = getCards();
+let currentIndex = cardList.length;
```

Comment:
We have already spoken about using `currentIndex` as a global variable. it is considered as a very bad practice